conda_build/skeletons/cran.py

```diff
@@ -1165,6 +1179,29 @@ def skeletonize(in_packages, output_dir=".", output_suffix="", add_maintainer=No
         if 'R' not in dep_dict:
             dep_dict['R'] = ''
 
+        os_type = cran_package.get("OS_type", '')
+        if os_type != 'unix' and os_type != 'windows' and os_type != '':
```

Comment:
I think you may use a ``if``  ``elif`` and this part can be the final ``else`` and I also think that ``print`` should be a ``log.warn`` maybe?
It will be something like:
```python
        if os_type == 'unix':
            d['skip_os'] = 'skip: True  # [not unix]'
            d["noarch_generic"] = ""
        elif os_type == 'windows':
            d['skip_os'] = 'skip: True  # [not win]'
            d["noarch_generic"] = ""
        elif os_type == '':
            d['skip_os'] = '# no skip'
        else:
            log.warn("Unknown OS_type: {} in CRAN package".format(os_type))
            d['skip_os'] = '# no skip'
            os_type = ''
```

Reply:
Adjusted logic. thanks